extensions/metrics/src/types.ts

```diff
@@ -20,11 +20,18 @@ export interface MetricsOptions {
     url: string;
     interval?: number;
   };
+
+  openApiSpec?: {
+    disabled?: boolean;
```

Comment:
Do you see possibility of having other options for `openApiSpec`? If not, let's use `openApiSpec?: boolean`.